docs/en/reference/limitations-and-known-issues.rst

```diff
@@ -130,10 +130,49 @@ included in the core of Doctrine ORM. However there are already two
 extensions out there that offer support for Nested Set with
 ORM:
 
-
 -  `Doctrine2 Hierarchical-Structural Behavior <https://github.com/guilhermeblanco/Doctrine2-Hierarchical-Structural-Behavior>`_
 -  `Doctrine2 NestedSet <https://github.com/blt04/doctrine2-nestedset>`_
 
+Using Traits in Entity Classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is no real "official" statement whether or to which extent traits
+are supported in entity or mapped superclasses.
+
+Traits were added in PHP 5.4 more than 10 years ago, but at the same time
+more than two years after the initial Doctrine 2 release and the time where
+core components were designed.
+
+In fact, this documentation mentions traits only in the context of
+:doc:`overriding field association mappings in subclasses <tutorials/override-field-association-mappings-in-subclasses>`. Also, the
+tests in the ORM codebase (as of writing) cover traits only in two (!) places,
+namely the aforementioned override and in an edge-case for the deprecated
+"Entity Generator".
```

Comment:
This paragraph is going to be hard to maintain: we will have to remember to remove this reference to "Entity Generator" when merging up to 3.0.x, and to change it any time we add any thing about traits in the docs. Maybe you should be more vague here?

Reply:
Changed that.